tests/main/fiddle-core-spec.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * @jest-environment node
+ */
+
+import { ElectronVersions, Runner } from '@electron/fiddle-core';
+import { WebContents } from 'electron';
+import { mocked } from 'jest-mock';
+
+import { setupFiddleCore, startFiddle } from '../../src/main/fiddle-core';
+import { ChildProcessMock } from '../mocks/child-process';
+import { ElectronVersionsMock, FiddleRunnerMock } from '../mocks/fiddle-core';
+import { WebContentsMock } from '../mocks/web-contents';
+
+jest.mock('@electron/fiddle-core', () => {
+  const { FiddleRunnerMock, InstallerMock } = require('../mocks/fiddle-core');
+
+  return {
+    Installer: InstallerMock,
+    Runner: FiddleRunnerMock,
+  };
+});
+
+describe('fiddle-core', () => {
+  let runner: FiddleRunnerMock;
+  let env: NodeJS.ProcessEnv;
```

Comment:
Can we make this somthing like `originalEnv`? There's a bunch of other `env`s elsewhere, this should make it slightly easier to tell them apart when skimming over the code